doc/howto/cross-network-inet-multicast-howto.md

```diff
@@ -0,0 +1,129 @@
+# OpenWeave + Happy Cross Network Multicast Inet Layer HOWTO
+
+You have decided that you would like to experiment with cross-network multicast using OpenWeave. This how-to guide illustrates using the Happy network simulation tool to set up a virtual topology that demonstrates using OpenWeave's Inet layer across two distinct networks with a multicast proxy.
+
+## Step 1: Download and Build mcproxy
+
+The IPv6 multicast proxy daemon, _mcproxy_, serves to proxy / forward / route IPv6 multicast traffic across two disjoint IPv6 network links.
+
+```
+% git clone https://github.com/mcproxy/mcproxy.git mcproxy
+% sudo apt-get install qt5-qmake qt5-default
+% cd mcproxy/mcproxy
+% qmake
+% make
+```
+
+## Step 2: Download, Build, and Install happy
+
+```
+% git clone https://github.com/openweave/happy.git happy
+% sudo apt-get install bridge-utils python-lockfile python-psutil python-setuptools
+% cd happy
+% sudo make install
+```
+
+## Step 3: Download and Build openweave-core
+
+```
+% git clone https://github.com/openweave/openweave-core.git openweave-core
+% cd openweave-core
+% ./configure
+% make
+```
+
+## Step 4: Establish the Happy Topology
+
+This creates a topology very similar to the [Happy Codelab](https://codelabs.developers.google.com/codelabs/happy-weave-getting-started/#0).
+
+```
+% happy-network-add ThreadNetwork thread
+% happy-network-address ThreadNetwork fd00:0000:0000:0006::
+% happy-network-add WiFiNetwork wifi
+% happy-network-address WiFiNetwork fd00:0000:0000:0001::
+% happy-network-address WiFiNetwork 192.168.1.0
+% happy-node-add ThreadNode
+% happy-node-add WiFiNode
+% happy-node-add BorderRouter
+% happy-node-join ThreadNode ThreadNetwork
+% happy-node-join WiFiNode WiFiNetwork
+% happy-node-join BorderRouter ThreadNetwork
+% happy-node-join BorderRouter WiFiNetwork
+% happy-network-route --prefix fd00:0000:0000:0006:: ThreadNetwork BorderRouter
+% happy-network-route --prefix fd00:0000:0000:0001:: WiFiNetwork BorderRouter
+% happy-network-route --prefix 192.168.1.0 WiFiNetwork BorderRouter
+```
+
+## Step 5: Create the mcproxy Configuration for the Happy Topology
+
+This creates an IPv6 multicast proxy between the simulated "Thread" network interface on "wpan0" and the simulated "WiFi" network interface on "wlan0" in the Happy topology we created above in Step 4.
+
+```
+% cat > mcproxy.conf << EOF
+protocol MLDv2;
+pinstance myProxy: wpan0 ==> wlan0;
+EOF
+```
+
+## Step 6: Run the Demonstration
```

Comment:
I was able to successfully run the demo.

For this section, I would clarify that the commands will be done in a separate terminal window for each node (Border Router, Receiver, Sender).

Reply:
Added comments with that guidance.

Reply:
Please take a look.

Reply:
LGTM